src/AbstractNav.js

```diff
@@ -27,9 +27,7 @@ class AbstractNav extends React.Component {
     activeKey: PropTypes.any,
   };
 
-  static defaultProps = {
-    as: 'ul',
-  };
+  static defaultProps = {};
```

Comment:
remove the empty object stubs?

Reply:
Yeah, was unsure about that, I can definitely remove them!